# Message with all currently detected stations

Header                  header          # Header containing timestamp etc. of this message
DetectedStation[]      stations         # All stations that are presented in the environment
